tests/compas/robots/test_model.py

```diff
@@ -52,6 +52,14 @@ def ur5():
                  ]
                  )
 
+@pytest.fixture
+def programatic_robot_model():
+    robot = RobotModel("robot")
+    link0 = robot.add_link("link0")
+    link1 = robot.add_link("link1")
+    robot.add_joint("joint1", Joint.CONTINUOUS, link0, link1)
+    link2 = robot.add_link("link2")
+    robot.add_joint("joint1", Joint.CONTINUOUS, link1, link2)
```

Comment:
could you add an `assert` based on the `robot.iter_chain()` (or other any of the other chain-related iterations) output to ensure that the kinematic chain is being rebuilt correctly on each addition?